odoo_module_migrate/migration_scripts/migrate_120_130.py

```diff
@@ -1,33 +1,36 @@
 # Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo_module_migrate.base_migration_script import BaseMigrationScript
 
-_TEXT_ERRORS = {
-    "*": {
-        "web_settings_dashboard":
-        "[V13] Reference to 'web_settings_dashboard'"
-        ". This module has been removed."
-    },
-    ".py": {
-        r".*@api.returns.*\n":
-        "[13] Use of deprecated decorator '@api.returns'",
-        r".*@api.cr.*\n":
-        "[13] Use of deprecated decorator '@api.cr'",
-        r".*@api.model_cr.*\n":
-        "[13] Use of deprecated decorator '@api.model_cr'",
-    },
-}
 
-_TEXT_REPLACES = {
-    ".py": {
-        r".*@api.multi.*\n": "",
-        r".*@api.one.*\n": "",
-        r"\.sudo\((?P<user>[^/)]+?)\)": r".with_user(\g<user>)",
-        r"\.suspend_security": ".sudo",
-        r"\"base_suspend_security\",\n": "",
-    },
-    ".xml": {
-        r"( |\t)*<field name=('|\")view_type('|\")>.*</field>\n": "",
-        r"src_model": "binding_model",
+class MigrationScript(BaseMigrationScript):
+    _TEXT_ERRORS = {
```

Comment:
Can you keep original indent somehow? Otherwise you may lose some changes during conflict resolving. Plus, all existing pull requests becomes in complicated conflict

E.g.

```
_TEXT_REPLACES = {
....
}

class MigrationScript(BaseMigrationScript):
    _TEXT_REPLACES = dict(**{
        # place for new lines
    }, **_TEXT_REPLACES)

```

Reply:
Hmmm, in my opinion it's ok how it is right now.
All current PRs will need to be rebased anyway

Reply:
If we keep it this way, we need to merge this PR before any others to avoid changes losses